keep/providers/vectordev_provider/vectordev_provider.py

```diff
@@ -32,33 +44,47 @@ def validate_config(self):
         )
 
     def _format_alert(
-        event: list[dict], provider_instance: "BaseProvider" = None
+        event: dict, provider_instance: "BaseProvider" = None
     ) -> AlertDto | list[AlertDto]:
         events = []
-        # event is a list of events
-        for e in event:
-            event_json = None
-            try:
-                event_json = json.loads(e.get("message"))
-            except json.JSONDecodeError:
-                pass
-
-            events.append(
+        if isinstance(event, list):
+            events = event
+        else:
+            events = [event]
+        try:
+            alert_dtos = []
+            for e in events:
+                event_type = e["keep_source_type"]
```

Comment:
I think we can make this more elegant.

Split it 2 to different foramtting functions and check the first event in events. If it has the event_type attribute (i.e., keep_source_type), we can assume that it's a provider we can ingest, otherwise, just add the event like it used to before.

wdyt?